lib/pages/login_signup_page.dart

```diff
@@ -1,342 +1,342 @@
-import 'package:flutter/material.dart';
-import 'package:hoopr/services/authentication.dart';
+// import 'package:flutter/material.dart';
```

Comment:
What happened here? :o

Reply:
This page isn't being used but throws errors due to other file changes so I commented them out for now. We can delete this file when cleaning up the code.